test/segment-loader.test.js

```diff
@@ -833,7 +807,8 @@ QUnit.module('SegmentLoader', function(hooks) {
           done();
```

Comment:
It won't let me write on the other line, but may be worth updating the test message above.